Bottom: 121b0cc2570d4d9dc2d9e3f4716f691078d76e1d
Top:    c9521c52ede2c7fce839bf16ab300ff498d3b95b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-18 02:22:38 +0000

Just make a black background.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index a4749b0..5b48957 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1933,9 +1933,9 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
 #endif
 
   glEnable (GL_STENCIL_TEST);
-  glClearColor (port->offlimits_color.red / 65535.,
-                port->offlimits_color.green / 65535.,
-                port->offlimits_color.blue / 65535.,
+  glClearColor (port->bg_color.red / 65535.,
+                port->bg_color.green / 65535.,
+                port->bg_color.blue / 65535.,
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
@@ -2038,6 +2038,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* Drawing operations as masked to areas where the stencil buffer is '0' */
   /* glStencilFunc (GL_GREATER, 1, 1); */           /* Draw only where stencil buffer is 0 */
 
+#if 0
   if (global_view_2d) {
     glBegin (GL_QUADS);
     glVertex3i (0,             0,              0);
@@ -2068,6 +2069,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     }
     glEnd ();
   }
+#endif
 
   ghid_draw_bg_image ();
